ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -70,3 +72,42 @@ def test_torch_cat(
         dim=unique_idx,
         out=None,
     )
+
+
+# permute
+@given(
+    dtype_values_axis=helpers.dtype_values_axis(
```

Comment:
Also, dims is supposed to be a tuple of ints, so the data generated for axis should be a tuple instead of an axis(integer)

Reply:
ok I added a ret_tuples to the helpers.get_axis function; this should only return a tuple now when turned on;